src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -384,9 +384,16 @@ protected function loadCacheDriver($cacheName, $objectManagerName, array $cacheD
 
         if (!isset($cacheDriver['namespace'])) {
             // generate a unique namespace for the given application
-            $env = $container->getParameter('kernel.root_dir').$container->getParameter('kernel.environment');
-            $hash = hash('sha256', $env);
-            $namespace = 'sf2'.$this->getMappingResourceExtension().'_'.$objectManagerName.'_'.$hash;
+            if ($container->hasParameter('cache.prefix.seed')) {
+                $seed = '.'.$container->getParameterBag()->resolveValue($container->getParameter('cache.prefix.seed'));
+            } elseif ($container->getParameter('kernel.debug')) {
+                $seed = '_'.$container->getParameter('kernel.root_dir');
```

Comment:
What do you think about reintroducing `kernel.root_dir` but only when `kernel.debug` is true, so that locally running apps have separate cache namespace by default?